fedimint-load-test-tool/src/main.rs

```diff
@@ -774,11 +774,11 @@ async fn get_gateway_public_key(
             cmd!(GatewayClnCli, "info").out_json().await
         }
     }?;
-    let node_pub_key = gateway_json["federations"][0]["registration"]["node_pub_key"]
+    let gateway_pub_key = gateway_json["federations"][0]["registration"]["gateway_pub_key"]
         .as_str()
         .ok_or_else(|| anyhow!("Missing node_pub_key field"))?;
```

Comment:
```suggestion
        .context("Missing gateway_pub_key field")?;
```
And change the import to `use anyhow::{bail, Context};`